commit be853349a4a9784eac564fec0467ad68595e617f
Author: Ganesh Goudar <ganeshgr@chelsio.com>
Date:   Wed Jul 4 17:49:33 2018 +0530

    cxgb4: Fix the condition to check if the card is T5
    
    [ Upstream commit dfecc759e64b0ea581468fe2359836f1998deac9 ]
    
    Use 'chip_ver' rather than 'chip' to check if the card
    is T5.
    
    Fixes: e8d452923ae6 ("cxgb4: clean up init_one")
    Signed-off-by: Ganesh Goudar <ganeshgr@chelsio.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
index a8926e97935e..c5d387be6cfe 100644
--- a/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
+++ b/drivers/net/ethernet/chelsio/cxgb4/cxgb4_main.c
@@ -5705,7 +5705,7 @@ static int init_one(struct pci_dev *pdev, const struct pci_device_id *ent)
 		if (t4_read_reg(adapter, LE_DB_CONFIG_A) & HASHEN_F) {
 			u32 hash_base, hash_reg;
 
-			if (chip <= CHELSIO_T5) {
+			if (chip_ver <= CHELSIO_T5) {
 				hash_reg = LE_DB_TID_HASHBASE_A;
 				hash_base = t4_read_reg(adapter, hash_reg);
 				adapter->tids.hash_base = hash_base / 4;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1116_be853349a4a9784eac564fec0467ad68595e617f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1116_be853349a4a9784eac564fec0467ad68595e617f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
